tests/test_getting_started.sh

```diff
@@ -33,7 +33,7 @@ curl $ENDPOINT_URL/chat/completions \
     -H "Content-Type: application/json" \
     -d '{
     "model": "gpt2",
-    "messages": [{"role": "assistant", "content": "You are a helpful assistant."}, {"role": "user", "content": "Hello!"}],
+    "messages": [{"role": "user", "content": "Hello!"}],
```

Comment:
Are the previous messages a chat sequence? why changed?

Reply:
Chat model must alternate user/assistant/user/assistant/...

Reply:
@KepingYan could you check if this is the correct sequence? I am not sure why we need to change it.